kadi/commands/states.py

```diff
@@ -1102,9 +1102,13 @@ def update_sun_vector_state(cls, date, transitions, state, idx):
             current index into transitions
         """
         if state["pcad_mode"] == "NPNT":
-            q_att = Quat([state[qc] for qc in QUAT_COMPS])
-            state["pitch"] = ska_sun.pitch(q_att.ra, q_att.dec, date)
-            state["off_nom_roll"] = ska_sun.off_nominal_roll(q_att, date)
+            ra, dec, roll = state["ra"], state["dec"], state["roll"]
+            time = date2secs(date)
+            sun_ra, sun_dec = ska_sun.position(time)
```

Comment:
I think maybe this should just be explicit about the position method it uses instead o using the ska_sun default.

Reply:
This is an intentional and important part of the design to delegate the behavior to `ska_sun.conf.sun_position_default_method`. That way you can choose to compute states with the fast position method.

Reply:
Fair.  I wasn't clear from our previous discussion (where accurate was getting faster) if kadi would still need to use both position methods in kadi.commands.states .  Should it always be accurate when we're doing validation?  I don't see that configured if so.

Reply:
Validation should run using the defaults just like a user or application normally would. That's also an important part of the design, that validation matches the way the code gets used.

If for some reason there was a code change that updated the ska_sun default, then all kinds of tests would break so that won't happen accidentally.